vm-allocator/src/system.rs

```diff
@@ -62,7 +62,7 @@ impl SystemAllocator {
         let page_size = pagesize() as u64;
         Some(SystemAllocator {
             io_address_space: if let (Some(b), Some(s)) = (io_base, io_size) {
-                Some(AddressAllocator::new(b, s, Some(0x400))?)
+                Some(AddressAllocator::new(b, s, Some(0x4))?)
```

Comment:
This should be `None`. IO address space is byte aligned.

Reply:
If you use `None`, it will default to `4`, right?

Reply:
@sboeuf Right. The setting here has been changed to 0x01. :)